mirgecom/initializers.py

```diff
@@ -462,6 +490,228 @@ def exact_rhs(self, discr, q, t=0.0):
         return flat_obj_array(massrhs, energyrhs, momrhs)
 
 
+class MultiLump:
+    r"""Solution initializer for multi-species N-dimensional Gaussian lump of mass.
```

Comment:
`Lump` itself also allows multiple species. How is this different? Why do we need both? The docs should say.

Reply:
The attempt to extend Lump to support multi-component flows was abandoned in favor of MultiLump, I just failed to rip out the remnants.

Reply:
Ripping done.

Reply:
Could you explain why extending the `Lump` for multi-species was difficult?

Reply:
The two initializers are not related, except that they both use a Gaussian.  I am unsure why I would want to create a relationship through documentation?

Reply:
Mostly because people will read the names (which are similar) in the docs and will wonder what the difference is.

Reply:
Indeed. I think the current docs help, but I can add more if needed.  I don't think anyone would use these outside the tests that use them.